src/Layout/ErrorBoundary/index.tsx

```diff
@@ -55,50 +65,137 @@ export class ErrorBoundary extends React.PureComponent<Props, State> {
       error,
       errorInfo,
     });
+
+    if (this.testResourceNotFound(error)) {
+      this.setState({
+        shouldReload: true,
+      });
+      this.startCountdown();
+    }
+  }
+
+  public componentWillUnmount(): void {
+    this.toDispose.dispose();
+  }
+
+  private testResourceNotFound(error: Error): boolean {
+    return /loading chunk [\d]+ failed/i.test(error.message);
```

Comment:
I wonder if we can merge this info with `refreshing banner` under View details, not to make users panic in such a pretty regular case.

Reply:
I'm not sure the banner fits here because this is the runtime error that breaks the UD, while the banner is used when the UD is functional and the user needs to be notified that something happened.